### PR TITLE
Persist per node reward rate

### LINKED ISSUE #3908: Persist node reward rate from NodeStakeUpdate transactions
### Problem

The latest protobuf for `NodeStakeUpdateTransactionBody` replaced the total reward rate with individual node's reward rate.  Mirrornode needs to update accordingly:

1. Remove `reward_sum` from table `node_stake`
2. Remove the reward sum calc logic in `NodeStackUpdateTransactionHandler`
3. Persist the individual node's reward rate

### Solution

refer to the description

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR adopts the changes in protobuf to persist per node staking reward rate.

- Remove `reward_sum` from table `node_stake`
- Persist per node staking `reward_rate`

**Related issue(s)**:

Fixes #3908

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — DISMISSED** [review]
LGTM besides an extra repository that can be removed

File: pom.xml [hunk 1]
```diff
@@ -134,6 +134,14 @@
...
+            <id>proto-staging</id>
+            <name>Protobufs staging</name>
+            <url>https://oss.sonatype.org/content/repositories/comhederahashgraph-1489</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
         <repository>
             <id>ossrh-snapshots</id>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
[the remark this answers is not in the document]
Comment (xin-hedera) [reply pom.xml:147]: perhaps it's worth leaving it as is so in the next snapshot cycle we don't have to add it back.

[the remark this answers is not in the document]
Comment (steven-sheehy) [reply pom.xml:147]: Less concerned about the cost of adding back vs accidentally retrieving the wrong artifact

[the remark this answers is not in the document]
Comment (xin-hedera) [reply pom.xml:147]: ic I'll remove it then

**Review by steven-sheehy — APPROVED** [review]

**Review by edwin-greene — APPROVED** [review]
Looks good.

**Pull request merged**